Cargo.toml

```diff
@@ -18,10 +18,13 @@ env_logger = "0.7"
 futures = "0.3"
 dirs = "3.0.1"
 tokio-socks = "0.5"
-reqwest = { version = "0.11", default-features = false, features = ["socks"] }
+reqwest = { version = "0.11", features = ["socks"] }
 chrono = "0.4"
 clap = { version = "4.3.19", features = ["derive"] }
 bitcoind = {version = "0.32", features = ["25_0"] }
+libtor = "47.13.0"
+socks = "0.3.4"
```

Comment:
we use tokio-socks. Do we need this dep also?

Reply:
We can remove it now... I was looking into socks earlier, but tokio-socks works well..